arealite/impl/dataset/VLdataset_dic.py

```diff
@@ -0,0 +1,140 @@
+from typing import Dict, List, Union
```

Comment:
The file names are somehow confusing. May you make it more meaningful and follow snake_case?